src/OpenLoco/src/GameCommands/VehicleRefit.cpp

```diff
@@ -0,0 +1,102 @@
+#include "Economy/Expenditures.h"
+#include "Entities/EntityManager.h"
+#include "GameCommands.h"
+#include "Objects/CargoObject.h"
+#include "Objects/ObjectManager.h"
+#include "Objects/VehicleObject.h"
+#include "Types.hpp"
+#include "Ui/WindowManager.h"
+#include "Vehicles/Vehicle.h"
+#include <OpenLoco/Interop/Interop.hpp>
+#include <OpenLoco/Utility/Numeric.hpp>
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    // 0x0042F6DB
+    static uint32_t vehicleRefit(const VehicleRefitArgs& args, const uint8_t flags)
+    {
+        setExpenditureType(ExpenditureType::TrainRunningCosts);
+
+        try
```

Comment:
is this standard in openloco? what are we expecting to throw an exception here?

Reply:
Creating a Vehicle can throw an exception, yeah.